trino/client.py

```diff
@@ -869,82 +869,79 @@ def map(self, value) -> Optional[float]:
         return float(value)
 
 
-class TemporalValueMapper():
-    def _get_number_of_digits(self, column):
-        args = column['arguments']
-        if len(args) == 0:
-            return 3, 0
-        ms_size = args[0]['value']
-        if ms_size == 0:
-            return -1, 0
-        ms_to_trim = ms_size - min(ms_size, 6)
-        return ms_size, ms_to_trim
-
-
-class TimeValueMapper(ValueMapper[time], TemporalValueMapper):
-    def __init__(self, column):
-        pattern = "%H:%M:%S"
-        ms_size, ms_to_trim = self._get_number_of_digits(column)
-        if ms_size > 0:
-            pattern += ".%f"
-        self.pattern = pattern
-        self.time_size = 9 + ms_size - ms_to_trim
+def round_millis(value: str, precision: int) -> str:
+    if len(value) > precision:
+        return str(round(Decimal(value) / Decimal(10 ** (len(value) - precision))))
```

Comment:
It seems to perform half-odd rounding while docs say `round` does half-even. (https://docs.python.org/3/library/functions.html#round)

> if two multiples are equally close, rounding is done toward the even choice

```python
In [18]: round(Decimal('2.5'))
Out[18]: 2

In [19]: round(Decimal('3.5'))
Out[19]: 4
```

```python
In [23]: round_millis('1234585', 6)
Out[23]: '123458'

In [24]: round_millis('1234575', 6)
Out[24]: '123458'
```

Also what if the value rounds up past 1 second? Receiver needs to perform the carry.

e.g. `round_millis('9999999', 6) -> '1000000'`.